mission_control/navigator_launch/launch/subsystems/sensors.launch

```diff
@@ -7,4 +7,7 @@
 
   <!-- Sick -->
   <include file="$(find navigator_launch)/launch/subsystems/sick_lidar.launch"/>
+  
+  <!-- Host Monitoring -->
+  <node name="host_monitor" pkg="navigator_gui" type="host_monitor.py">
```

Comment:
I think this should actually be in master shore launch. Whoever runs network kill then will also be checking that everything is connected. We don't want the boat running this, because we won't be able to see what is connected if the boat is not.